src/translate/translate.tsx

```diff
@@ -85,28 +86,38 @@ const getTranslationSetGracefully = (
   msgid: string
 ) => {
   const supportedLanguages = Object.keys(translations).filter((l) => {
-    const [userLocale] = locale.split('_')
-    const [supportedLocale] = l.split('_')
-    return userLocale === supportedLocale
+    const userLocale = parseLocale(locale)
```

Comment:
you could use this cool fn `isSameLocale` here

Reply:
(if modified, so maybe not worth)

Reply:
It would also compare the country code, but what we're trying to make is a list of all `locale*` eg. all `fr, fr_fr` etc. should all be matched but if I change it, it would do direct matches only.